src/browserosaurus.json

```diff
@@ -1,7 +1,8 @@
 {
   "version": "<%= version %>",
   "settings": {
-    "configMode": "override"
+    "configMode": "override",
+    "autoOrdering": true
```

Comment:
Thanks for adding this functionality. I’m not sure the key is descriptive though. Actually I’m wondering if we include it at all. I think the browsers should always be in the order of the config, and have a dev rule that the default set is always alphabetical. One less config to worry about. I think this is why I’m really not bothered about “merge” mode. I mean if a user has gone to the trouble of making their own config then they know what they want.